Backend/src/main/java/idealab/api/operations/PrintJobOperations.java

```diff
@@ -56,21 +51,16 @@ public PrintJobResponse newPrintJob(PrintJobNewRequest printJobNewRequest, Princ
         //Integer employeeId = printJobNewRequest.getEmployeeId();
         LocalDateTime currentTime = LocalDateTime.now();
 
-        // TODO: Hash email so it is not in plaintext!!
-        String emailHash = printJobNewRequest.getEmail();
-        EmailHash databaseEmail = emailHashRepo.findByEmailHash(emailHash);
-        
-        if (databaseEmail == null) {
-            databaseEmail = new EmailHash(emailHash);
-            databaseEmail = emailHashRepo.save(databaseEmail);
-        }
-
-        CustomerInfo customer = customerInfoRepo.findByEmailHashId(databaseEmail);
+        CustomerInfo customer = customerInfoRepo.findByEmail(email);
         if (customer == null) {
-            customer = new CustomerInfo(databaseEmail, customerFirstName, customerLastName, email);
+            customer = new CustomerInfo(null, customerFirstName, customerLastName, email);
             customer = customerInfoRepo.save(customer);
         }
 
+        if(customer.getPrintJobs() != null && customer.getPrintJobs().size() >= 5) {
```

Comment:
This looks great. They are allowed to have more than 5 open print jobs, they just aren't allowed to request more than 5 in one day. I think this logic needs to add if they have 5+ open print jobs that were opened today

Reply:
We will address this separately to merge in the email hash changes.